include/dfu/suit_dfu_fetch_source.h

```diff
@@ -65,37 +72,38 @@ typedef int (*suit_dfu_fetch_source_request_fn)(const uint8_t *uri, size_t uri_l
 /**
  * @brief Register fetch source
  *
- * @param[in]   request_fn		Fetch source callback function. The suit processor will
- * iterate over registered callback functions when fetching from an URI is needed.
- * Attention! Order of execution of callback functions is not guaranteed!
- *
+ * @param[in] request_fn  Fetch source callback function. The suit processor will iterate over
+ *                        registered callback functions when fetching from an URI is needed.
+ *                        Attention! Order of execution of callback functions is not guaranteed!
```

Comment:
Don't we have standard doxygen commands to avoid inventions like "Attention!" or "Danger ahead!" ;)